Updated the docs to include annotaion info in the steps (#80745)
openshift/openshift-docs: modules/jira-configuring-acs.adoc

.Procedure
. In the {product-title-short} portal, go to *Platform Configuration* -> *Integrations*.
. Scroll down to the *Notifier Integrations* section and select *Jira Software*.
. Click *New integration*.
. Enter a name for *Integration name*.
. Enter the user credentials in the *Username* and *Password or API token* fields.
. For *Issue type*, enter a valid link:https://confluence.atlassian.com/adminjiracloud/issue-types-844500742.html[Jira Issue Type], for example *Task*, *Sub-task*, or *Bug*.
. Enter the Jira server URL in the *Jira URL* field.
. Enter the key of the project in which you want to create issues in the *Default project* field.

> . Optional: Use the *Annotation key for project* field to create issues in different Jira projects. You can use annotations to dynamically create issues. To do this:

Reviewer: ```suggestion
. Optional: Use the *Annotation key for project* field to create issues in different Jira projects by completing the following steps. You can use annotations to dynamically create issues.
```

.. Add an annotation similar to the following example in your namespace or deployment YAML file, where `jira/project-key` is the annotation key that you specify in your Jira integration. You can create an annotation for the deployment or the namespace.
+
[source,yaml,subs="+quotes"]
----
annotations:
  jira/project-key: _<jira_project_key>_

> ----

Reviewer: ```suggestion
[source,yaml,subs="+quotes"]
----
annotations:
# ...
  jira/project-key: <jira_project_key>
# ...
```

Reviewer: As per this [guideline](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#signs-and-symbols-in-code-blocks).
